src/main/java/com/amazon/opendistroforelasticsearch/security/configuration/ClusterInfoHolder.java

```diff
@@ -111,14 +112,14 @@ public Boolean hasNode(DiscoveryNode node) {
     }
 
     private static boolean clusterHas6xNodes(ClusterState state) {
-        return state.nodes().getMinNodeVersion().before(Version.V_7_0_0);
```

Comment:
Are we using Version anywhere else in file? If not lets remove it from import

Reply:
No, it's not used anywhere else in this file.

Reply:
Sure. It'll be taken care of in one of future PRs.